src/compat/dbtype.h

```diff
@@ -429,6 +437,15 @@
 
 #define DB_GET_ENUM_COLLATION(value) db_get_enum_collation(value)
 
+#define DB_GET_COMPRESSED_STRING(value) db_get_compressed_string(value)
```

Comment:
Please consider adding a macro variant (to generate inline code). See similar  DB_PULL_STRING or DB_GET_STRING_SAFE.